NJekyll/site/docs/lang/nodejs-iojs.md

```diff
@@ -157,6 +157,11 @@ This configuration will produce a build with 4 jobs for all combinations of node
     io.js 1.0.x      x64
 
 
+## Deploying a Node.js website to Azure
+
+You can use Web Deploy to [deploy your Node website](/site/images/docs/deployment/web-deploy.md#Using-web-deploy-with-a-node.js-website) to Azure.
```

Comment:
NOTE: Not sure if this link is correct/accurate. It's trying to link to the new section i added on the other document.